src/utils/regex.ts

```diff
@@ -0,0 +1,3 @@
+export const regexEscape = (input: string) => input.replace(/[/\-\\^$*+?.()|[\]{}]/g, '\\$&');
+
+export const regexName = (name: string) => regexEscape(name).replaceAll(/(е|ё)/g, '(е|ё)');
```

Comment:
In this case for replacing will get only lower case letters
May be add the insensitive flag?

Reply:
added